bundles/AdminBundle/Resources/public/js/pimcore/asset/helpers/gridConfigDialog.js

```diff
@@ -304,8 +308,8 @@ pimcore.asset.helpers.gridConfigDialog = Class.create(pimcore.element.helpers.gr
                 }
             ];
 
-            var languagestore = [["",t("default")],["none",t("none")]];
-            for (var i = 0; i < pimcore.settings.websiteLanguages.length; i++) {
+            var languagestore = [];
```

Comment:
It should be possible to unset the language for some columns i.e. authorName
![image](https://user-images.githubusercontent.com/5137917/88158131-f0825000-cc0b-11ea-8810-4a19bb89da77.png)